src/server.h

```diff
@@ -672,6 +673,9 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 	std::unordered_map<std::string, ModMetadata *> m_mod_storages;
 	float m_mod_storage_save_timer;
 
+	// CSM flavour limits byteflag
+	u64 m_csm_flavour_limits = CSMFlavourLimit::CSM_FL_NONE;
```

Comment:
this is default value for object constructor, no default minetest.conf value which doesn't exists yet (will do it when other parts will be finished)